implementation/pom.xml

```diff
@@ -60,6 +60,11 @@
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.9</version>
+        </dependency>
```

Comment:
Can we do this without pulling in this dep ?

Reply:
Fair point, I've pushed an update that replaces the Apache Commons call with a quick and dirty helper method.